webapp/src/features/intersections/reports/report-list-table.tsx

```diff
@@ -157,7 +114,7 @@ export const ReportListTable = (props: ReportListTableProps) => {
           {
             <TableBody>
               {reports.map((report: ReportMetadata) => (
-                <ReportRow report={report} />
+                <ReportRow report={report} onViewReport={onViewReport} />
```

Comment:
suggestion(non-blocking): define a key for the ReportRow. 
```suggestion
                <ReportRow key={report.reportName} report={report} onViewReport={onViewReport} />
```

From [Sonarqube](https://rules.sonarsource.com/typescript/impact/maintainability/RSPEC-6477/): 

> To optimize the rendering of React list components, a unique identifier (UID) is required for each list item. This UID lets React identify the item throughout its lifetime. To provide it, use the key attribute of the list item. When the key attribute is missing, React will default to using the item’s index inside the list component. If the element ordering changes, it will cause keys to not match up between renders, recreating the DOM. It can negatively impact performance and may cause issues with the component state.